### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by bajajcodes**
- Responsive navigation bar on a media-query breakpoint enables hamburger menu for navigation. 
- Navbar supports `navbar-brand` for project name, project logo or both, `navbar-nav` for navigation.

https://user-images.githubusercontent.com/29247011/153083631-b029bd92-2e9e-408a-8b2f-c139692a4648.mp4

### CODE REVIEW

File: css/components/navbar.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+.hamburger{
+    background-color: transparent;
+    border: 0;
+    font-size: var(--size-10);
+    cursor: pointer;
+    display: none;
+}
+
+.hamburger:focus{
+    outline: none;
```
Comment (junaidshaikh-js) [inline css/components/navbar.css]: This will make the hamburger unaccessible to the keyboard users.

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

**Review by AnuragSahu11 — COMMENTED** [review]
Use outline: 2px solid transparent; outline-offset: 2px;
Instead of outline none. Other than that looks good.

File: css/components/navbar.css (hunk 1 above)
Comment (bajajcodes) [reply css/components/navbar.css]: Then how to make the hamburger accessible to the keyboard users.

**Pull request merged**